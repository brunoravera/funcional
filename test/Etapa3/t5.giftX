
//ejercicio que testea filtroFV
::ej1::
son {VERDADERO}
//

//ejercicio que testea filtroFV
::ej2::
todas {VERDADERO} {VERDADERO} {V}
//

//ejercicio que testea filtroFV
::ej3::
verdaderas {=si ~nono}
//
