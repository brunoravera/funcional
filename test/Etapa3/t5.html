<html>
	<head>
		<link rel="stylesheet" href="gift.css">
	</head>
	<body>
<div class="giftComentario">
ejercicio que testea filtroFV
</div>
<span class="giftNombre">
ej3</span>
<div class="giftFragmento">
verdaderas</div>


<div class="giftMO">
\item <div class="giftFragmento">
si</div>


\item <div class="giftFragmento">
nono</div>


</div>

<div class="giftFragmento">
</div>


<div class="giftComentario">

</div>

	</body>
</html>
