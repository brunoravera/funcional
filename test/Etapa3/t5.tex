\documentclass{gift}
\begin{document}
\begin{giftComentario}
ejercicio que testea filtroFV
\end  {giftComentario}
\giftNombre{ej3}
\begin{giftFragmento}
verdaderas\end  {giftFragmento}


\begin{giftMO}
\item \begin{giftFragmento}
si\end  {giftFragmento}


\item \begin{giftFragmento}
nono\end  {giftFragmento}


\end  {giftMO}

\begin{giftFragmento}
\end  {giftFragmento}


\begin{giftComentario}

\end  {giftComentario}

\end{document}
