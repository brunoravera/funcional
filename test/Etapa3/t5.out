CUESTIONARIO
============

PREGUNTA ej3
COMENTARIO INICIAL: ejercicio que testea filtroFV
[Q [TXT "verdaderas"],A (MO [OK [TXT "si"],NOK [TXT "nono"]]),Q [TXT ""]]
COMENTARIO FINAL: 
------------
